test/functional/tests/ci/test_ci_write_mode_write_around.py

```diff
@@ -0,0 +1,212 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+import time
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Unit, Size
+from test_tools.dd import Dd
+from test_tools.iostat import IOstatBasic
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.CI()
+def test_ci_write_mode():
+    """
+    title: Verification test for write mode: write around
+    description: Verify if write around mode works as expected and caches only reads
+    and does not cache write
+    pass criteria:
+    - writes are not cached
+    """
+
+    with TestRun.step("Prepare devices"):
+        cache_device = TestRun.disks["cache"]
+        core_device = TestRun.disks["core"]
+
+        cache_device.create_partitions([Size(1024, Unit.MebiByte)])
+        core_device.create_partitions([Size(2048, Unit.MebiByte)])
+
+        cache_device = cache_device.partitions[0]
+        core_device = core_device.partitions[0]
+
+    with TestRun.step("Test read from cache"):
+        count = 100
+        block_size = 4096
+        selected_cache_modes = [casadm.CacheMode.WT, casadm.CacheMode.WA]
+
+        for mode in selected_cache_modes:
+            print("------Start CAS Linux in " + mode.value + " mode")
+            # This is where the real test starts
+            cache = casadm.start_cache(cache_dev=cache_device, cache_id=1, force=True)
+            casadm.add_core(cache, core_device)
+            time.sleep(2.5)
+            casadm.set_cache_mode(mode, 1)
+
+            dd = (
+                Dd()
+                .input("/dev/cas1-1")
+                .output("$TMP_DIR/file001.bin")
+                .count(count)
+                .block_size(Size(1, Unit.Blocks4096))
+                .iflag("direct")
+                .skip(10000)
+            )
+            dd.run()
+
+            iostat = IOstatBasic.get_iostat_list([cache_device.parent_device])
+            read_cache_1 = iostat[0].total_reads.value
+
+            dd = (
+                Dd()
+                .input("/dev/cas1-1")
+                .output("$TMP_DIR/file001.bin")
+                .count(count)
+                .block_size(Size(1, Unit.Blocks4096))
+                .iflag("direct")
+                .skip(10000)
+            )
+            dd.run()
+
+            iostat = IOstatBasic.get_iostat_list([cache_device.parent_device])
+            read_cache_2 = iostat[0].total_reads.value
+
+            read_cache_delta = (int(read_cache_2) - int(read_cache_1))
+            data_read = count * block_size
```

Comment:
Why not use `Size(count, Unit.Blocks4096)`? No conversions in the line above would be needed.